recipes/xerces-c/all/conandata.yml

```diff
@@ -1,17 +1,17 @@
 sources:
-  "3.2.2":
-    url: "https://github.com/apache/xerces-c/archive/v3.2.2.tar.gz"
-    sha256: "7fe5af7d7ad9d4a06503c15fb5bb0aa5f2ba7959700d16c21b8bd183ca542e7f"
   "3.2.3":
     url: "https://github.com/apache/xerces-c/archive/v3.2.3.tar.gz"
     sha256: "a7cf582d618c048ef6a4684457a641940179c446e5f02c81f582f5952755a76a"
-patches:
   "3.2.2":
+    url: "https://github.com/apache/xerces-c/archive/v3.2.2.tar.gz"
+    sha256: "7fe5af7d7ad9d4a06503c15fb5bb0aa5f2ba7959700d16c21b8bd183ca542e7f"
```

Comment:
I'm not sure of the usefulness to reorder conandata.yml, especially considering [qchateau's bot](https://qchateau.github.io/conan-center-bot/#/updatable) uses the [reverse order](https://github.com/qchateau/conan-center-index/commit/2e584de07b623ccd00cbfa8f71697e494d0816af)

Reply:
the order could be arbitrary in YAML. no need to constantly re-order the things. just insert a new version to the top, or to the bottom, or to the middle.

Reply:
The bot doesn't use reverse order, it honors whatever order already used in the yml file. Maybe it's reverse order if there is only one version. Anyway I disagree with this default order, I think it's easier for maintenance to see last version first, and to have one convention (a mix of different versions conventions in one library can make the release order hard to undestand).

Reply:
As long as it's a direction, and not random, it could be merged. Hopefully that's easy to agree on 😆 

Given the evolution and the number of recipes... I am willing to concede on a uniform ordering though it would be a nice to have 🤷